pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -84,7 +84,7 @@
 , # sniprun dependencies
   bashInteractive
 , coreutils
-, curl
+, curl # also aw-watcher-vim dependency
```

Comment:
This doesn't matter, as curl is used in a few other dependencies as well

Reply:
Oh okay, I've removed it then!